### PR TITLE
Fixes #12880 - red hat repositories page not showing

### PR DESCRIPTION
**Pull request by johnpmitsch**
The version of jquery we use was upgraded in the rails 4 migration. ```.live``` was deprecated and so was ```browser.msie```

### CODE REVIEW

File: app/assets/javascripts/katello/widgets/filtertable/filtertable.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var filtertable = (function() {
             var theTable = $('table.filter_table');
             var filter = $('#filter');
 
-            filter.live('change, keyup', function(){
+            $('body').on('change, keyup', filter,  function(){
                 $.uiTableFilter(theTable, this.value);
             });
```
Comment (jlsherrill) [inline app/assets/javascripts/katello/widgets/filtertable/filtertable.js]: this file can probably be deleted

File: app/assets/javascripts/katello/widgets/one_panel.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ var one_panel = {
 
     });
 
-    $('.block').live('click', function(e)
+    $('body').on('click', '.block', function(e)
     {
         var activeBlock = $(this);
```
Comment (jlsherrill) [inline app/assets/javascripts/katello/widgets/one_panel.js]: this file can probably be deleted

File: app/assets/javascripts/katello/common/env_select.js [hunk 3]
```diff
@@ -93,10 +93,10 @@ var env_select =   {
...
-        $('#path-expanded').live('click', env_select.close);
-        $('.path_link').live('click', env_select.env_selected);
-        $('.path_entry').live('click', env_select.path_selected);
+        $('body').on('click', '#path-collapsed', env_select.expand);
+        $('body').on('click', '#path-expanded', env_select.close);
+        $('body').on('click', '.path_link', env_select.env_selected);
+        $('body').on('click', '.path_entry', env_select.path_selected);
 
         //If we mouse over the entries box, deselect what is already selected
         $('#path-entries').mouseover(env_select.disable_active);
```
Comment (jlsherrill) [inline app/assets/javascripts/katello/common/env_select.js]: this file can probably be deleted

**Comment by johnpmitsch** [comment]
@jlsherrill updated

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**